docs/start/authentication/api-accounts.mdx

```diff
@@ -79,10 +79,10 @@ App-level API accounts do not come pre-configured with an access token. Each tim
 
 There is no way to manually revoke or force-regenerate app-level API account access tokens. However, either of the following actions triggers a token refresh:
 
-* When the store owner's email address changes
+* When the store owner's email address changes 
```

Comment:
The ticket mentions store-level API accounts, but doesn't mention app-level API accounts. Just wanted to confirm that this line didn't need changing

Reply:
@piotrekkaminski Since you approved, I'll assume that yes,  this line doesn't need changing